### DISCUSSION TITLE
Forward Text Message | Doesn't appear on phone but appear on Web WhatsApp

### DISCUSSION THREAD

**Discussion by 1337-sys** [tulir/whatsmeow#147] [WhatsApp protocol Q&A]
Hi,

When I forward a text message(no problem image, video, document), it does not appear on phone, but on web. Where am I doing wrong?

```
waProto.ContextInfo{
		ForwardingScore: proto.Uint32(1),
		IsForwarded:     proto.Bool(true),
	}
```

**Comment by 1337-sys** [comment]
Phone is iPhone

**Comment by alexdsgmoura** [comment]
same problem

**Comment by docxplusgmoon** [comment]
I want to forward message. Could you share your full code for forwarding message?

**Comment by idan-dagan-imagenai** [comment]
Can someone please show to to forward image from one group to another?? can you please provide code snippet??
